modules/analytics/analytics.js

```diff
@@ -123,6 +123,13 @@ class Analytics {
                     permanentProperties.group = group;
                 }
 
+                if (window.jitsiRegionInfo) {
```

Comment:
Can we put this into `window.JitsiMeetJS.regionInfo` instead? A little while ago we settled on that as our only global object (we are not there yet, but new code should follow the rules! 👮 ) See: https://github.com/jitsi/jitsi-meet/blob/master/modules/util/helpers.js#L50